configure.ac

```diff
@@ -1843,6 +1843,17 @@ else
     pm_names=""
 fi
 #
+
+PAC_SET_HEADER_LIB_PATH(pmix)
+if test "$with_pmix" != "no"; then
+    # disable built-in PMI
+    with_pmi="no"
```

Comment:
@pavanbalaji do you think we should also disable built-in process managers when linking against PMIx? They won't work correctly, so probably best to avoid confusion.